go/vt/vttablet/tabletmanager/vdiff/engine.go

```diff
@@ -153,12 +153,12 @@ func (vde *Engine) openLocked(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+
```

Comment:
The vdiff engine's controller list is initialized as empty in Open and I think that we should do the same for the stats so that they stay in sync and the stats don't grow and grow with old cruft. We can do that by placing this block just after the `if len(vde.controllers) > 0 {` block above:
```
	func() {
		globalStats.mu.Lock()
		defer globalStats.mu.Unlock()
		globalStats.controllers = make(map[int64]*controller)
	}()
```